COM-421 Patient search return list is showing precise location and no…

prashantsable1991 opened the pull request.

prashantsable1991 committed:
COM-421 Patient search return list is showing precise location and not Quartier

tmvumbi2 commented:
@prashantsable1991 Please update the filter's label to read `Village` (EN) or `Quartier/ Villages` (FR)

![image](https://user-images.githubusercontent.com/22587098/60887632-22b94d80-a255-11e9-8986-0ae44d57296f.png)

prashantsable1991 committed:
COM-421 updated the filter's label on registration search page

tmvumbi2 committed:
COM-421 Update the label for placeholder of search address field

Replaces 'Area' by 'Village' because we changed the field's label for the address filter
